core/src/main/java/io/kestra/core/serializers/FileSerde.java

```diff
@@ -78,10 +93,54 @@ public static boolean reader(BufferedReader input, int maxLines, Consumer<Object
     }
 
     private static Object convert(String row) throws JsonProcessingException {
-        return MAPPER.readValue(row, TYPE_REFERENCE);
+        return DEFAULT_OBJECT_MAPPER.readValue(row, DEFAULT_TYPE_REFERENCE);
     }
 
     private static <T> T convert(String row, Class<T> cls) throws JsonProcessingException {
-        return MAPPER.readValue(row, cls);
+        return DEFAULT_OBJECT_MAPPER.readValue(row, cls);
+    }
+
+    public static Flux<Object> readAll(InputStream in) throws IOException {
+        return readAll(DEFAULT_OBJECT_MAPPER, in, DEFAULT_TYPE_REFERENCE);
+    }
+
+    public static <T> Flux<T> readAll(InputStream in, TypeReference<T> type) throws IOException {
+        return readAll(DEFAULT_OBJECT_MAPPER, in, type);
+    }
+
+    public static Flux<Object> readAll(ObjectMapper objectMapper, InputStream in) throws IOException {
+        return readAll(objectMapper, in, DEFAULT_TYPE_REFERENCE);
+    }
+
+    public static <T> Flux<T> readAll(ObjectMapper objectMapper, InputStream in, TypeReference<T> type) throws IOException {
+        return Flux.generate(
+            () -> createMappingIterator(objectMapper, in, type),
+            throwBiFunction((iterator, sink) -> {
+                final T value = iterator.hasNextValue() ? iterator.nextValue() : null;
+                Optional.ofNullable(value).ifPresentOrElse(sink::next, sink::complete);
+                return iterator;
+            }),
+            throwConsumer(MappingIterator::close)
+        );
+    }
```

Comment:
Is it needed to allow passing an ObjectMapper?
The same for the corresponding writeAll method.